vagrant_resources/host_vms/docker_host

```diff
@@ -0,0 +1,39 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+btrfs_volume = 'vagrant_resources/host_vms/auxiliary_disks/docker_host_btrfs_volume.vdi'
+
+Vagrant.configure("2") do |config|
+  config.vm.provider :virtualbox do |provider, override|
+    provider.name = "docker_host"
+    override.vm.box = "ubuntu/xenial64"
+    provider.customize ['modifyvm', :id, '--nictype1', 'virtio']
+    provider.customize ['modifyvm', :id, '--nictype2', 'virtio']
+    provider.memory = 2048
+    unless File.exist?(btrfs_volume)
+      provider.customize ['createhd', '--filename', btrfs_volume, '--size', 120 * 1024]
+    end
+    provider.customize ['storageattach', :id,  '--storagectl', 'SCSI', '--port', 2, '--device', 0, '--type', 'hdd', '--medium', btrfs_volume]
+  end
+
+  config.vm.define  "docker_host"
+
+  config.vm.provision "shell", inline:
```

Comment:
Can you consolidate some of these? Some like https://stackoverflow.com/questions/32787547/is-it-possible-to-send-multiple-commands-to-vagrant-ssh-via-a-shell-script